src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -486,9 +489,11 @@ def rename_nucbases(structure):
     Returns:
 
     """
-    chainresdic = dict([(c.get_id(), [r.get_resname() for r in c.get_residues()]) for m in structure for c in m])
+    chainresdic = dict([(c.get_id(),
+                       [r.get_resname() for r in c.get_residues()]) for m in structure for c in m])
 
-    nucleotide_list = ["CYT", "C", "DC", "THY", "T", "DT", "ADE", "A", "DA", "G", "GUA", "DG", "U", "URI"]
+    nucleotide_list = ["CYT", "C", "DC", "THY", "T", "DT", "ADE",
+                       "A", "DA", "G", "GUA", "DG", "U", "URI"]
```

Comment:
Be careful here: "U" is also used for Uranium Ion (see [https://rascar.science.uu.nl/haddock2.4/library](https://rascar.science.uu.nl/haddock2.4/library)).
Even tho, rarely used...

Reply:
Maybe this is something we should talk with Alex, we should probably make changes for uranium instead of the nucleic acid base in my opinion.